rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/90_sparse_vector.yml

```diff
@@ -0,0 +1,143 @@
+---
+"Indexing and searching sparse vectors":
```

Comment:
Added a test that checks indexing and performs a simulated text_expansion query.